src/Language/PureScript/Docs/Prim.hs

```diff
@@ -258,12 +258,25 @@ kindSymbol = primType "Symbol" $ T.unlines
   , ""
   , "Construct types of this kind using the same literal syntax as documented"
   , "for strings."
+  , ""
+  , "    type Hello :: Symbol {- String -}"
+  , "    type Hello = \"Hello, world\""
+  , ""
   ]
 
 kindRow :: Declaration
 kindRow = primType "Row" $ T.unlines
   [ "`Row` is the kind constructor of label-indexed types which map type-level strings to other types."
-  , "For example, the kind of `Record` is `Row Type -> Type`, mapping field names to values."
+  , "For example..."
+  , ""
+  , "     type ClosedRow = ( label :: TypeName, name :: String )"
+  , "     type OpenRow otherRows = ( label :: TypeName, name :: String | otherRows )"
+  , "     type FinalRow = OpenRows ( anotherLabel :: Int )"
+  , "     -- FinalRow = ( label :: TypeName, name :: String, anotherLabel :: Int )"
+  , ""
+  , "Rows can be 'open' or 'closed'. 'Closed' rows cannot be extended with additional label-type pairs."
+  , "'Open' rows can be extended."
```

Comment:
I object to this way of explaining rows. An ‘open row’ isn't a type in PureScript's type system; all row types are closed, and ‘open rows’ are just a particular pattern for using type synonyms to create closed rows. Someone recently [got tripped up over this](https://discourse.purescript.org/t/partially-applied-type-in-proxy-with-row-type-param/2996), thinking that an open row was something they could pass around in a `Proxy`, which a reasonable thing to think if we say that rows are types and open rows are rows.

Reply:
Honestly, I didn't know this either. I'm now wondering why I never thought of passing an open row around via a Proxy...

Reply:
I've pushed a commit that drops this.

On another hand, row kinds are technically `label-type` pairs, which means one can have a row of symbols:
```
type Foo = ( label :: "label", int :: 4 )
```
Should we include this there? Or leave it out because its too advanced and is better documented elsewhere?

Reply:
One actually can't do that particular thing! Rows need to be kind-homogeneous; you can't use a `Symbol` with one label and an `Int` with another in the same row—the former would create a `Row Symbol` and the latter a `Row Int`.

Some slight rewording of the explanation of the kind of `Row` might clear the above up, and I think *that* would be more useful than a simple demonstration of the technique (which doesn't have obvious applications for most, I would say).

Reply:
:man_facepalming: Ugh. I actually know this, but wasn't thinking.

> Some slight rewording of the explanation of the kind of Row might clear the above up, and I think that would be more useful than a simple demonstration of the technique (which doesn't have obvious applications for most, I would say).

Feel free to propose an improved rendering.